### PR TITLE
[ADD] elarning REPO + PSC

### PR DESCRIPTION
**Pull request by etobella**
@OCA/board This is the PSC and Repo necessary for the elearning platform

### CODE REVIEW

File: conf/psc/elearning.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+elearning-maintainers:
```
Comment (simahawk) [inline conf/psc/elearning.yml:1]:
```suggestion
e-learning-maintainers:
```

File: conf/repo/elearning.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+elearning:
```
Comment (simahawk) [inline conf/repo/elearning.yml:1]:
```suggestion
e-learning:
```

File: conf/psc/elearning.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  members:
+    - etobella
+  name: Elearning maintainers
+  representatives:
+    - etobella
+elearning-maintainers-psc-representative:
```
Comment (simahawk) [inline conf/psc/elearning.yml:7]:
```suggestion
e-learning-maintainers-psc-representative:
```

**Comment by etobella** [comment]
Sure! Comments attended @simahawk

**Review by simahawk — APPROVED** [review]

**Review by vdewulf — APPROVED** [review]
It's all good for me, thanks!

**Pull request merged**